src/web/crate_details.rs

```diff
@@ -42,6 +42,7 @@ pub struct CrateDetails {
     github_stars: Option<i32>,
     github_forks: Option<i32>,
     github_issues: Option<i32>,
+    // TODO: Is this even needed for rendering pages?
     pub(crate) metadata: MetaData,
```

Comment:
I think what my comment may have been getting at is the `pub target_name: String,` field, which is (maybe?) `default_target` since the `pub(crate) doc_targets: Vec<String>,` field also exists

Reply:
IIRC `target_name` is the name of the library (which can be different from the crate).